hackathon_site/event/models.py

```diff
@@ -38,5 +38,11 @@ class Profile(models.Model):
     created_at = models.DateTimeField(auto_now_add=True, null=False)
     updated_at = models.DateTimeField(auto_now=True, null=False)
 
+    def save(self, *args, **kwargs):
```

Comment:
Is this called by Django automatically? I don't see you calling it anywhere in the model

Reply:
This will be called when you do things like

```python
profile = Profile(**kwargs)
profile.save()
```
or
```python
profile = Profile.objects.create(**kwargs)
```

There are a whole host of things that will call either of these methods, like `.create()` or `.update()` for serializers. The notable exceptions are [`bulk_create()`](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#bulk-create) and [`bulk_update()`](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#bulk-update), which we won't be using for this model.